Jenkinsfile.release

```diff
@@ -0,0 +1,79 @@
+#!groovy
+
+node {
+    properties([
+        parameters([
+            string(
+                defaultValue: '',
+                description: 'SHA of an existing container image. Usually what\'s currently on staging.', 
```

Comment:
I'd also consider revising this text so that it is specific about requiring the first 7 characters of the relevant Git SHA.